installers/ansible/roles/pgo-operator/files/crds/perconapgclusters-crd.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: perconapgclusters.crunchydata.com
+spec:
+  group: crunchydata.com
```

Comment:
pg.percona.com

Reply:
I think we should do this in separate PR after merge